ansible/roles/bastion-lite/tasks/k8s.yml

```diff
@@ -0,0 +1,19 @@
+---
+- tags:
+    - bastion_k8s
+  block:
+    - name: Add epel-release repo
```

Comment:
This will need to change. No more EPEL. You seem to have PIP3 installed anyway. Might as well use that?

Reply:
This only runs if install_k8s_modules is set to true, which it isn't for this config by default. I left it in for backwards compatibility.

And yes, I'm planning on pip and version pinning where possible.

Reply:
and you don't have to convince me no more EPEL :) I'm not using it _anywhere_ in this config.